reporters_db/data/reporters.json

```diff
@@ -6566,7 +6566,14 @@
                 "us;federal"
             ],
             "name": "Federal Communications Commission Reports",
-            "variations": {}
+            "variations": {
+
+                "FCC 2d": "F.C.C.2d",
+                "FCC": "F.C.C."
+        }
+                "FCC 2d": "F.C.C.2d",
+                "FCC" : "F.C.C."
+        }
```

Comment:
This appears to be a typo.  The following should fix it and the tests.

```
"variations": {
                "FCC": "F.C.C.",
                "FCC 2d": "F.C.C.2d"
            }
```

Reply:
This is also why the tests are failing.

Reply:
Yes thanks. I will clean that up.

Reply:
This was also supposed to be part of different PR and got merged here do to my not know how Github forks work. Sorry it won't happen again!